### ISSUE TITLE
Update get-pipenv.py to use up-to-date pip

### ISSUE THREAD

**Issue by and-semakin** [pypa/pipenv#4265]
Links:
* [`get-pipenv.py`](https://github.com/pypa/pipenv/blob/master/get-pipenv.py);
* [documentation describing its usage](https://pipenv.pypa.io/en/latest/install/#crude-installation-of-pipenv).

The issue is that `get-pipenv.py` at the moment contains `pip==9.0.1` which is a little bit outdated. New releases of `pip` contain a lot of bug fixes (see [changelog](https://pip.pypa.io/en/stable/news/#id331)).

I understand that this script is used only once to bootstrap the system, but anyway it's better to use an up-to-date version.

If you think that it's worth updating, then I would be happy to provide a PR.

**Comment by frostming** [comment]
I had a look at `get-pip.py` and realized that it can be used to install other packages than `pip`, `setuptools` and `wheel`.

Can you please try the following command to see if it works?
```console
$ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
$ python get-pip.py pipenv
```

If so, we just remove that file and update the documentation.

**Comment by frostming** [comment]
AFAICT Upgrading pip can't resolve the problem, it lacks `setuptools` to building sdist-only packages. `get-pip.py` works because `pip` is a self-contained package without other dependencies. However `pipenv` depends on a sdist-only package `distlib`.

I marked it as low priority but I would be happy if anyone can help.

**Comment by and-semakin** [comment]
I confirm that the approach with `get-pip.py` works perfectly on the distributions with `pip` and `setuptools` already installed. Also, I confirm that neither `get-pip.py pipenv` nor `get-pipenv.py` work on a distribution without `setuptools`.

I've posted my `Dockerfile` on which I test here: https://github.com/pypa/pipenv/issues/1092#issuecomment-633258304

**Comment by oz123** [comment]
`get-pip.py` is created in: https://github.com/pypa/get-pip

**Comment by oz123** [comment]
Fixed in #4920 .
